src/Dictionaries/HierarchyDictionariesUtils.h

```diff
@@ -429,18 +432,19 @@ template <typename KeyType>
 ColumnPtr getKeysDescendantsArray(
```

Comment:
But they key considered as found if it exists in parent_to_child map, so maybe parent_keys_found?

Reply:
@azat I am not sure, maybe valid_keys is better because we pass keys, parent_to_child map and valid_keys are keys that are valid in parent_to_child map.